### PR TITLE
Update test implementation

### PR DESCRIPTION
**Pull request by stepanLav**
That PR:
- Change logic to find elements in the app by testID
- Update test implementation in order to decrease repetition

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "strict": true,
     "target": "es2022",
     "module": "es2022",
-    "lib": ["es2022"],
+    "lib": ["es2022", "dom"],
```
Comment (stepanLav) [inline tsconfig.json:13]: Added in order to find indexedDB object in tests/system/utils/interactWithDatabase.ts, can't pass eslint rules

File: src/renderer/pages/Onboarding/Welcome/WelcomeCard.tsx [hunk 2]
```diff
@@ -9,13 +9,15 @@ type Props = {
   iconName: IconNames;
   onClick?: () => void;
   disabled?: boolean;
+  buttonTestId?: string;
```
Comment (johnthecat) [inline src/renderer/pages/Onboarding/Welcome/WelcomeCard.tsx]: Why not simple `testId`?

File: src/renderer/shared/ui-entities/AccountExplorer/AccountExplorers.tsx [hunk 3]
```diff
@@ -20,7 +21,12 @@ export const AccountExplorers = memo(({ accountId, chain, children }: Props) =>
   return (
     <Popover align="end" dialog testId="AccountExplorers">
       <Popover.Trigger>
-        <IconButton name="details" className="text-icon-default" onClick={(e) => e.stopPropagation()} />
+        <IconButton
+          name="details"
+          className="text-icon-default"
+          testId={TEST_IDS.MAIN.INFO_BUTTON}
```
Comment (johnthecat) [inline src/renderer/shared/ui-entities/AccountExplorer/AccountExplorers.tsx]: I think ui-entities and ui-kit should be free from app-centric constants like this. It's better to get testId from component props and have reasonable default, usually it's component name

**Review by johnthecat — CHANGES_REQUESTED** [review]

File: src/renderer/pages/Onboarding/Welcome/WelcomeCard.tsx (hunk 2 above)
Comment (stepanLav) [reply src/renderer/pages/Onboarding/Welcome/WelcomeCard.tsx]: WelcomeCard has many elements, I thought that it may need to have a few ids, but now only one is needed, I will change it, ty!

File: src/renderer/shared/constants/testIds.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    NOVA_WALLET_BUTTON: 'onboarding-nova-wallet-button',
+    WALLET_CONNECT_BUTTON: 'onboarding-wallet-connect-button',
+    LEDGER_BUTTON: 'onboarding-ledger-button',
+    WALLET_NAME_INPUT: 'onboarding-wallet-name-input',
+    WALLET_ADDRESS_INPUT: 'onboarding-wallet-address-input',
+  },
+  ASSETS: {
+    SETTINGS_WIDGET: 'assets-settings-widget',
+  },
+  MAIN: {
```
Comment (tuul-wq) [inline src/renderer/shared/constants/testIds.ts]: As I understand here we locate common items? If so let's name is as `common`

File: src/renderer/shared/constants/testIds.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    WALLET_BUTTON: 'main-wallet-button',
+    BACK_BUTTON: 'main-back-button',
+    CONTINUE_BUTTON: 'main-continue-button',
+    INFO_BUTTON: 'main-info-button',
+  },
+  TRANSFER: {
```
Comment (tuul-wq) [inline src/renderer/shared/constants/testIds.ts]: Maybe just `operations`?
`Network fee` is a really common thing for any kind of operation
Thus I suggest to not go deeper that 1 level of nesting, or we'll end up with file similar to `en.json` which is hard to parse visually.

File: src/renderer/shared/ui-entities/AccountExplorer/AccountExplorers.tsx (hunk 3 above)
Comment (stepanLav) [reply src/renderer/shared/ui-entities/AccountExplorer/AccountExplorers.tsx]: Is that what you meant?
https://github.com/novasamatech/nova-spektr/pull/2668/commits/d6a12e55d588fc3eb8b699293ed50b70a3df73e8

File: src/renderer/shared/constants/testIds.ts (hunk 4 above)
Comment (stepanLav) [reply src/renderer/shared/constants/testIds.ts]: https://github.com/novasamatech/nova-spektr/pull/2668/commits/97b462c05a7e92381ef0de87eba4ef253a4342a8

File: src/renderer/shared/constants/testIds.ts (hunk 5 above)
Comment (stepanLav) [reply src/renderer/shared/constants/testIds.ts]: https://github.com/novasamatech/nova-spektr/pull/2668/commits/d6a12e55d588fc3eb8b699293ed50b70a3df73e8

**Review by tuul-wq — APPROVED** [review]

**Review by johnthecat — APPROVED** [review]
